src/sentry/utils/pytest/fixtures.py

```diff
@@ -442,3 +443,8 @@ def inner(key, value):
         request.addfinalizer(lambda: ctx_mgr.__exit__(None, None, None))
 
     return inner
+
+
+@pytest.fixture
+def django_cache():
+    cache.clear()
```

Comment:
Will this break other tests when multiple tests run in parallel?

Reply:
AttributeError: 'BoundLoggerFilteringAtNotset' object has no attribute 'log'